src/Mod/Fem/Gui/DlgSettingsFemGeneralImp.cpp

```diff
@@ -43,7 +43,7 @@ DlgSettingsFemGeneralImp::DlgSettingsFemGeneralImp(QWidget* parent)
     ui->cmb_def_solver->clear();
     std::vector<std::string> Solvers = {"None"};
 
-    if (!Fem::Tools::checkIfBinaryExists("CCX", "ccx", "ccx").empty()) {
+    if (!Fem::Tools::checkIfBinaryExists("Ccx", "ccx", "ccx").empty()) {
```

Comment:
just curious about the reasoning behind this change?

Reply:
The first parameter is used to get the parameter path `auto paramPath = "User parameter:BaseApp/Preferences/Mod/Fem/" + prefSection;`.

I found the config for calculix wasn't applied to the UI component(button always hidden) when testing, and it turned out that the section string didn't match the one in the config schema, `<FCParamGroup Name="Ccx"/>`, which was revealed by directly inspecting `user.cfg`.

I didn't bother to locate where the schema is defined. Instead I just made the change and gave it a test.